### PR TITLE
Create Code of Conduct.md

### PR DESCRIPTION
**Pull request by scarlet-tobar**
Add Code of Conduct to Florence Governance

### CODE REVIEW

**Review by 1011X — APPROVED** [review]

**Comment by mal0ki** [comment]
@skrlet13 can you update this PR once we finalize the CoC and ping us for review then?

**Comment by scarlet-tobar** [comment]
@maloki Sure

**Comment by scarlet-tobar** [comment]
Updated the commit

File: Code of Conduct.md [hunk 1]
```diff
@@ -1,160 +1,143 @@
...
+'''If you believe anyone is in physical danger, please notify appropriate law enforcement first.''' If you are unsure what law enforcement agency is appropriate, please include this in your report and we will attempt to notify them.
 
 We encourage you to report incidents, even if you are unsure whether the incident is a violation, or whether the space where it happened is covered by this Code of Conduct. We would much rather have a few extra reports where we decide to take no action, than miss a report of an actual violation. We do not look negatively on you if we find the incident is not a violation. Knowing about incidents that are not violations, or happen outside our spaces, can help us to improve the Code of Conduct and the processes surrounding it.
 
-## **What happens after you file a report?**
-
+## What happens after you file a report? 
 You will receive an email from the moderators acknowledging receipt. The current Working Group members are @1011X@mastodon.social, @jhaye@social.libre.fi and @skrlet13@chile.masto.host.
```
Comment (mal0ki) [inline Code of Conduct.md]: Update these handles to become url links to profiles, rather than mailto: emails

**Review by mal0ki — CHANGES_REQUESTED** [review]
This one thing needs to be changed.

File: Code of Conduct.md (hunk 1 above)
Comment (scarlet-tobar) [reply Code of Conduct.md]: sure

Comment (scarlet-tobar) [reply Code of Conduct.md]: Done!

**Comment by jhaye** [comment]
This seems pretty good already! I think we should retain the CC BY-SA licensing notice from the wiki. Same with https://github.com/florence-social/mastodon-fork/pull/122.

Probably just adding "This work is licensed under a [Creative Commons Attribution-ShareAlike 4.0 International License](http://creativecommons.org/licenses/by-sa/4.0/)."  should be enough.

**Review by jhaye — APPROVED** [review]

**Comment by mal0ki** [comment]
Since we've merged the new version elsewhere, I'm closing this PR

**Pull request closed without merging**